bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```
@@ -211,6 +247,9 @@ metadata:
               "image": "placeholder",
               "size": 2
             }
+          },
+          "status": {
+            "podStatus": null
```

Could you use a condition with an empty array also in this example?
```suggestion
            "conditions": []
```

---

Yes, if it's okay to keep `[]` to the `podStatus` filed then i will add there. but, we can replace the `podStatus` filed with `conditions` field. because the `podStatus` key  is added by referring the status field of `activemqartemis_types.go` file.